x/programs/rust/wasmlanche-sdk/src/program.rs

```diff
@@ -71,13 +92,13 @@ impl<K> Program<K> {
     /// # use wasmlanche_sdk::{types::Address, Program};
     /// #
     /// # let program_id = [0; Address::LEN];
-    /// # let target: Program<()> = borsh::from_slice(&program_id).expect("the program should deserialize");
+    /// # let target: Program<()> = borsh::from_slice(&program_id).unwrap();
     /// let increment = 10;
-    /// let params = borsh::to_vec(&increment).expect("failed to borsh serialize params");
+    /// let params = borsh::to_vec(&increment).expect("serialization error");
     /// let max_units = 1000000;
-    /// target
-    ///     .call_function("call_with_param", &params, max_units)
-    ///     .unwrap()
+    /// let has_incremented: bool = target.call_function("increment", &params, max_units)?;
+    /// assert!(has_incremented);
+    /// # Ok::<(), wasmlanche_sdk::ExternalCallError>(())
```

Comment:
My reasoning for these is that the borsh deserialization and serialization above are infaillible. So it was a nice opportunity to return an error from the example.